presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -123,6 +172,28 @@ public TableScanNode tableScan(List<Symbol> symbols, Map<Symbol, ColumnHandle> a
         );
     }
 
+    public TableScanNode tableScan(String tableName, Map<String, String> symbolMap)
+    {
+        QualifiedObjectName name = new QualifiedObjectName(session.getCatalog().get(), session.getSchema().get(), tableName);
+        TableHandle tableHandle = metadata.getTableHandle(session, name).orElse(new TableHandle(
```

Comment:
or `orElseThrow`.

However, I'm not convinced you need real tables for the sake of plan rule testing. You just need to instantiate `TableScanNode` with some predetermined arguments. The more explicit they are the better, so please consider not using session and metadata here.